--- pyppeteer/chromium_downloader.py	2021-08-17 12:07:22.089216008 -0400
+++ pyppeteer/chromium_downloader.py	2021-08-17 12:09:44.821605860 -0400
@@ -68,14 +68,9 @@
     """Download data from url."""
     logger.warning('Starting Chromium download. ' 'Download may take a few minutes.')
 
-    # Uncomment the statement below to disable HTTPS warnings and allow
-    # download without certificate verification. This is *strongly* as it
-    # opens the code to man-in-the-middle (and other) vulnerabilities; see
-    # https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
-    # for more.
-    # urllib3.disable_warnings()
-
-    with urllib3.PoolManager() as http:
+    import certifi
+    
+    with urllib3.PoolManager(cert_reqs='CERT_REQUIRED', ca_certs=certifi.where()) as http:
         # Get data from url.
         # set preload_content=False means using stream later.
         r = http.request('GET', url, preload_content=False)
