From 7258242d6c8017f35e5d48c2f346eba9acb1da37 Mon Sep 17 00:00:00 2001
From: Sylvain Corlay <sylvain.corlay@gmail.com>
Date: Sat, 19 Sep 2020 13:05:16 +0200
Subject: [PATCH] conda-forge

---
 pyppeteer/chromium_downloader.py | 13 +++++--------
 1 file changed, 5 insertions(+), 8 deletions(-)

diff --git a/pyppeteer/chromium_downloader.py b/pyppeteer/chromium_downloader.py
index 69bf896..2e30942 100644
--- a/pyppeteer/chromium_downloader.py
+++ b/pyppeteer/chromium_downloader.py
@@ -72,17 +72,14 @@ def get_url() -> str:
 
 def download_zip(url: str) -> BytesIO:
     """Download data from url."""
-    logger.warning('start chromium download.\n'
+    logger.warning('start secure https chromium download.\n'
                    'Download may take a few minutes.')
-
-    # disable warnings so that we don't need a cert.
-    # see https://urllib3.readthedocs.io/en/latest/advanced-usage.html for more
-    urllib3.disable_warnings()
-
-    with urllib3.PoolManager() as http:
+    import certifi
+    with urllib3.PoolManager(cert_reqs='CERT_REQUIRED',
+                             ca_certs=certifi.where()) as https:
         # Get data from url.
         # set preload_content=False means using stream later.
-        data = http.request('GET', url, preload_content=False)
+        data = https.request('GET', url, preload_content=False)
 
         try:
             total_length = int(data.headers['content-length'])
-- 
2.32.0

